pkgs/build-support/build-fhs-chrootenv/env.nix

```diff
@@ -129,7 +129,7 @@ let
   setupLibDirs_multi = ''
     mkdir -m0755 lib32
     mkdir -m0755 lib64
-    ln -s lib32 lib
+    ln -s lib64 lib
```

Comment:
Wild guess: shoudn't this lib32 vs lib64 symlink be set depending on "isMultiBuild"?